implementation-reports/wordpress.md

```diff
@@ -0,0 +1,203 @@
+
+
+### Update Tests (3.1.3)
+
+SHOULD
+
+* [x] [Update Test #1](https://webmention.rocks/update/1)
+* [ ] [Update Test #2](https://webmention.rocks/update/2)
+
+#### Implementation Notes
+
+Doesn't pass update test #2 yet because it doesn't send webmentions to removed links. [Tracked in issue #76.](https://github.com/pfefferle/wordpress-webmention/issues/76#issuecomment-225306427)
+
+
+### Delete Tests (3.1.4)
+
+SHOULD
+
+* [ ] [Delete Test #1](https://webmention.rocks/delete/1)
+
+#### Implementation Notes
+
+WordPress returns HTTP 404 for deleted posts, not 410.
+
+
+### Security Considerations (4)
+
+* [ ] The sender avoids sending a Webmention to a loopback address (SHOULD)
+
+Deliberate design choice to support loopback addresses to enable local development and testing.
+
+
+### Extensions
+
+This implementation has also implemented the following extensions.
+
+* [x] [Salmention](http://indiewebcamp.com/Salmention)
+* [ ] [Vouch](http://indiewebcamp.com/Vouch)
+* [ ] Other: _______
+
+
+## Receiving
+
+Indicate which type of response the receiver provides:
+
+* [x] HTTP 200 - Receiver synchronously processes the Webmention request (not recommended)
+* [ ] HTTP 201 - Receiver creates a status URL the sender can use to check the status of the Webmention
+* [ ] HTTP 202 - Receiver processes the Webmention asynchronously
+
+Describe the response body (if any) which is returned in the request:
+
+Plain text permalink URL of the newly created comment on the target page.
+
+
+### Request Verification (3.2.1)
+
+* [x] Verifies source and target are valid URLs, rejecting with HTTP 400 (MUST)
+* [x] Verifies that target is a valid resource for which the receiver accepts Webmentions, rejecting with HTTP 400 (SHOULD)
+* [x] Ignores fragment when checking if target is supported (SHOULD)
+
+
+### Webmention Verification (3.2.2)
+
+* [ ] Verification is processed asynchronously (SHOULD)
+* [x] Follows at least one HTTP redirect on source URL (MUST)
+* [x] Respects a self-imposed limit on number of HTTP redirects to follow (MUST)
+
+Limits to 5 redirects. Returns 400 if that limit is exceeded.
+
+
+#### Source URL content-types supported
+
+Please list the content types that your implementation supports when checking if the source document links to the target URL.
+
+* [x] HTML
+* [x] Other: text/*, application/json, etc.
+
+
+### HTML Verification (3.2.2)
+
+The tests below apply when the source document is HTML.
+
+* [x] Accepts a Webmention where the target URL is in an `<a>` tag
+* [x] Accepts a Webmention where the target URL is in an `<img>` tag
+* [x] Accepts a Webmention where the target URL is in an `<video>` tag
+* [x] Accepts a Webmention where the target URL is in an `<audio>` tag
+* [ ] Rejects a Webmention where the target URL is in the document as text
+* [ ] Rejects a Webmention where the target URL is in an `<a>` tag inside an HTML comment
+* [ ] Rejects a Webmention where the target URL is not in the document
```

Comment:
are you sure it doesn't reject these? This is really the most important one to reject since it's an obvious source of spam